relay-server/src/endpoints/mod.rs

```diff
@@ -74,6 +75,7 @@ pub fn routes(config: &Config) -> Router<ServiceState>{
         .route("/api/:project_id/minidump/", minidump::route(config))
         .route("/api/:project_id/events/:event_id/attachments/", post(attachments::handle))
         .route("/api/:project_id/unreal/:sentry_key/", unreal::route(config))
+        .route("/api/:project_id/traces-data/", traces_data::route(config))
```

Comment:
Rename to `/traces/` to be consistent with https://opentelemetry.io/docs/languages/sdk-configuration/otlp-exporter/#otel_exporter_otlp_traces_protocol?

Reply:
Why is it currently "traces-data"?

Reply:
In the previous impl it was `/spans/` i wanted to name it after the data type it accepts (`TracesData`) to be consistent with `/minidump/`, `/csp-report/`, etc. But reading the link above a bit more, we should maybe even name it `/otel/v1/traces/` to be future proof.

> For OTLP/HTTP, exporters in the SDK construct signal-specific URLs when this environment variable is set. This means that if you’re sending traces, metrics, and logs, the following URLs are constructed from the example above:
>
> Traces: "http://my-api-endpoint/v1/traces"
> Metrics: "http://my-api-endpoint/v1/metrics"
> Logs: "http://my-api-endpoint/v1/logs"


https://opentelemetry.io/docs/languages/sdk-configuration/otlp-exporter/#otel_exporter_otlp_endpoint